data/mships.txt

```diff
@@ -4509,8 +4509,8 @@ ship "SKVS-4 Minnow"
 	explode "small explosion" 30
 	explode "medium explosion" 20
 	"final explode" "final explosion small"
-	description `A water tanker ship for transporting fresh water to nearby colonies or stations which are incapable of generating enough fresh water themselves. The Minnow itself is slow and lightly armed making it incapable of defending itself or escaping which makes it less ideal for travelling to frontier worlds alone with the raise of piracy.`
-	description `Now that most colony are developed enough to supply their own water these ships became much rarer as their need decreases. Now only a small number remains parked in systems that have stations for occasional resupply mission.`
+	description `A water tanker ship for transporting fresh water to nearby colonies or stations which are incapable of generating enough fresh water for themselves. The Minnow itself is slow and lightly armed, making it incapable of defending itself or escaping from any threat, which makes it less ideal for travelling to frontier worlds alone with the rise of piracy.`
+	description `	Now that most colonies are developed enough to make their own water, these ships have become much rarer as the need for them decreases. Only a small number remain parked in systems that have stations for occasional resupply missions.`
```

Comment:
```suggestion
	description `A water tanker ship for transporting fresh water to nearby colonies or stations which are incapable of generating enough fresh water for themselves. The Minnow itself is slow and lightly armed making it incapable of defending itself or escaping from any threat, which makes it less ideal for travelling to frontier worlds alone with the rise of piracy.`
	description `	Now that most colonies are developed enough to supply their own water, these ships have become much rarer as the need for them decreases. Only a small number remain parked in systems that have stations for occasional resupply mission.`
```
supply->make just change the meaning, they can get their water by other ways than making. Also "making water" doesn't sound exactly right nor economical in most case.